Gemfile

```diff
@@ -28,4 +28,10 @@ group :development, :test do
   gem 'byebug'
   gem 'web-console', '~> 2.0'
   gem 'spring'
+  gem 'sqlite3'
+
+  gem 'rspec-rails', '~> 3.0.0'
+  gem 'factory_girl_rails'
```

Comment:
The factory_girl_rails, database_cleaner and fake gems should be moved to the `group :test` block.

Please remove the blind and mocha gems from this block as they are no longer used.